syntax_and_semantics/new,_initialize_and_allocate.md

```diff
@@ -55,7 +55,7 @@ class Person
 end
 ```
 
-First, note the `self.new` notation. This means that the method belongs to the **class** `Person`, not to particular instances of that class. This is why we can do `Person.new`.
+First, note the `self.new` notation. This means is is a [class method](class_methods.md) and belongs to the **class** `Person`, not to particular instances of that class. This is why we can do `Person.new`.
```

Comment:
"This means it is a [class method](class_methods.md) and it belongs (...)"

Reply:
How about just "This is a class method (...)"